lib/src/Base/Geom/LevelSet.cxx

```diff
@@ -134,6 +139,26 @@ Bool LevelSet::contains(const Point & point) const
   return operator_(function_(point)[0], level_);
 }
 
+/* Compute the Euclidean distance from a given point to the LevelSet */
+Scalar LevelSet::computeDistanceToDomain(const Point & point) const
```

Comment:
Would it be useful to set a norm different from the Euclidian norm, e.g. 1-norm or infinity-norm?

Reply:
Perhaps. I do not think other norms are needed in the HSIC context, but we could add them in the future.